src/lib/TasksManager.ts

```diff
@@ -63,10 +71,13 @@ export class TasksManager {
   }
 
   async launch(): Promise<void> {
-    const browser = new Browser();
-    await browser.create();
+    const chromium = new Browser('chromium');
+    await chromium.create();
+    const firefox = new Browser('firefox');
+    await firefox.create();
```

Comment:
maybe an optimization in the future would be to have 2 image of renderscript, each with it's own worker, and the crawler chooses which worker to target based on the configuration, that way we keep the image the same size.
But it's definitely a lot more work and something we can do if this ends up being an issue.

Reply:
That would be perfect indeed! Let's keep it in mind.

And yes I'll add some tests with `firefox`!